common.yml

```diff
@@ -1020,6 +1057,13 @@ tasks:
         tool: mongotop
     - func: "create release artifacts"
 
+- name: sign
+  depends_on:
+    - name: dist
+  commands:
+    - func: "fetch dist release artifacts"
```

Comment:
once https://jira.mongodb.org/browse/TOOLS-2438 is done, https://jira.mongodb.org/browse/TOOLS-2440 will add another step here that actually signs the artifacts that need to be signed.